account-kit/react/src/components/auth/card/loading/oauth.tsx

```diff
@@ -0,0 +1,61 @@
+import { useCallback, useEffect } from "react";
+import { useSignerStatus } from "../../../../hooks/useSignerStatus.js";
+import { useAuthContext, type AuthStep } from "../../context.js";
+import { ContinueWithOAuth } from "../../../../icons/oauth.js";
+import { ConnectionError } from "../error/connection-error.js";
+import { capitalize } from "../../../../utils.js";
+import { useOAuthVerify } from "../../hooks/useOAuthVerify.js";
+
+interface CompletingOAuthProps {
+  authStep: Extract<AuthStep, { type: "oauth_completing" }>;
+}
+
+export const CompletingOAuth = ({ authStep }: CompletingOAuthProps) => {
+  const { isConnected } = useSignerStatus();
+  const { setAuthStep } = useAuthContext();
+  const { authenticate } = useOAuthVerify();
+  const restartFlow = useCallback(
+    () => authenticate(authStep.config),
+    [authenticate]
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useCallback has a missing dependency: 'authStep.config'. Either include it or remove the dependency array.
<!-- __reviewdog__:ChBlZTZhZTgxZGI4OTlmN2VmEgZyZGpzb24= -->

Reply:
LOOOL this has my name on them now, amazing. (I had to use one of my PATs because I think the alchemy-bot token was deleted at some point)